src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -473,6 +475,7 @@ namespace AppInstaller::CLI::Workflow
                         availableVersion,
                         sourceName
                         });
+                    availableUpgradesCount++;

```

Comment:
This availableUpgradesCount++ should technically move inside if (updateAvailabe) statement above.